### DISCUSSION TITLE
Lesson 0: failure to request faucet funds

### DISCUSSION THREAD

**Discussion by Ugochi-Eze** [smartcontractkit/full-blockchain-solidity-course-py#1521] [Q&A] [answered]
I made a tweet requesting faucet funds and copied url, pasted on rinkeby authenticated faucet but it fails to fund the rinkeby net account and shows faucet offline which I do not understand. Please what do I do next? Thanks.
Here is a picture of what it shows
![16539761586184498139437538771705](https://user-images.githubusercontent.com/106381272/171101876-d670652f-76a2-4461-86ab-89ec9275d4e3.jpg)

**Comment by cromewar** [comment, marked as answer]
Hello @Ugochi-Eze 
yes that faucet goes down time to time, I highly recommend to use Chainlink Faucet.
https://faucets.chain.link/

**Comment by Ugochi-Eze** [comment]
Thank you very much
